Data/Services/CustomerService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Data.Entities;
+using Data.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Data.Repositories
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly List<Customer> _customers = new();
+
+        public void Create(Customer createdCustomer)
+        {
+            bool emailAlreadyExists = _customers.Any(customer => customer.Email == createdCustomer.Email);
+            if (emailAlreadyExists)
+                throw new ArgumentException("Email already exists");
+
+            bool cpfAlreadyExists = _customers.Any(customer => customer.Cpf == createdCustomer.Cpf);
+            if (cpfAlreadyExists)
+                throw new ArgumentException("Cpf already exists");
+            
+
+            createdCustomer.Id = _customers.LastOrDefault()?.Id + 1 ?? 1; 
+
+            _customers.Add(createdCustomer);
+        }
+
+        public bool Delete(long id)
+        {
+            Customer customer = GetById(id);
+
+            return _customers.Remove(customer);
+        }
+
+        public List<Customer> GetAll()
+        {
+
+            return _customers;
+        }
+
+        public Customer GetById(long id)
+        {
+            var result = _customers.FirstOrDefault(customer => customer.Id == id);
+
+            if (result is null)
+                throw new ArgumentException($"Customer with id {id} not found");
+            
+
+            return result;
+        }
+
+        public bool Update(Customer updatedCustomer)
```

Comment:
```suggestion
        public void Update(Customer updatedCustomer)
```
Sugestão: Poderia ser um método void, pois não retorna nada. O método é utilizado para atualizar informações de um customer.